apps/codeforafrica/src/lib/data/blockify/posts.js

```diff
@@ -0,0 +1,29 @@
+import {
+  getPosts,
+  formatPost,
+  getTagsByPrimaryTag,
+} from "@/codeforafrica/lib/data/utils/posts";
+
+async function posts(block, api, context) {
+  const { primaryTag, labels, stories = {} } = block;
+  const { query } = context;
+  const { featured, title } = stories;
+  const featuredStory = featured && (formatPost(featured, primaryTag) || null);
+  const options = {
+    ...query,
+  };
+  const { posts: list, pagination } = await getPosts(api, options, primaryTag);
+  const tags = await getTagsByPrimaryTag(api, primaryTag);
+  return {
+    title,
+    labels,
+    tags,
+    featured: featuredStory || null,
+    posts: list,
```

Comment:
```suggestion
  const { posts, pagination } = await getPosts(api, options, primaryTag);
  const tags = await getTagsByPrimaryTag(api, primaryTag);
  return {
    title,
    labels,
    tags,
    featured,
    posts,
```

Why rename `posts` to `list` only to rename it back to `posts`?

Reply:
@kilemensi eslint complains as the function is already name posts

Reply:
... cool, add a single line comment explaining why you're renaming it then.